app/services/recent-events.ts

```diff
@@ -183,6 +274,21 @@ export class RecentEventsService extends StatefulService<IRecentEventsState> {
     return await fetch(new Request(url, { headers, body, method: 'POST' })).then(handleResponse);
   }
 
+  async readAlert(event: IRecentEvent) {
```

Comment:
This is a strangely named function but I guess it matches the API route so it's ok.